README.md

```diff
@@ -84,7 +84,10 @@ the material listed below.
     - [BGP Configuration](docs/bgp.md) explains how to manage the BGP peering
       for integration of a Calico cluster in your network
     - [External Connectivity](docs/ExternalConnectivity.md) describes how to
-      configure external connectivity for hosts on their own Layer 2 segment
+      configure external connectivity for hosts on their own Layer 2 segment
+    - [Calico As a Service](docs/CalicoAsService.md) describes how to run the
+      `calico/node` and `calico/node-libnetwork` images as system processes or
+      services with systemd examples.

```

Comment:
sentence doesn't soudn quite right:  as system processes or services with systemd examples.

Is there a missing comma?  Maybe make the "with systemd examples" a separate sentence.